### PR TITLE
Docs updated, Prerequisites section [21.0]

### PR DESCRIPTION
**Pull request by Karm**
SSIA
Tested also on on "clean-room" Ubuntu.

### CODE REVIEW

**Comment by Karm** [comment]
Huh...  @zakkak  I thought there was some blacklist for `*.md` for the GitHub Workflow? Is it expected for the CI to run on a docs update?

**Comment by zakkak** [comment]
> Huh... @zakkak I thought there was some blacklist for `*.md` for the GitHub Workflow? Is it expected for the CI to run on a docs update?

Nice catch. This is not the mandrel specific pipeline, it's the upstream graal one. We should probably fix it upstream :)

**Comment by Karm** [comment]
@zakkak Updated. Ready to be merged.

File: README-Mandrel.md [hunk 1]
```diff
@@ -35,16 +35,19 @@ Mandrel distributions can be downloaded from [the repository's releases](https:/
 Mandrel's `native-image` depends on the following packages:
 * glibc-devel
 * zlib-devel
+* freetype-devel
 * gcc
```
Comment (zakkak) [inline README-Mandrel.md:39]: Shouldn't we include `libstdc++-static` in this list?

**Review by zakkak — APPROVED** [review]
Thanks for the updates @Karm

**Comment by zakkak** [comment]
Closing as 21.1 will supersede 21.0 later this month

**Pull request closed without merging**